gallery/src/styles/NxTable/NxTableExamples.tsx

```diff
@@ -11,6 +11,8 @@ import { GalleryExampleTile } from '../../gallery-components/GalleryTiles';
 import NxTableClickableExample from './NxTableClickableExample';
 import NxTableErrorExample from './NxTableErrorStateExample';
 import NxTableFixedLayoutExample from './NxTableFixedLayoutExample';
+import NxTableFilterExample from '../../components/NxTable/NxTableFilterExample';
+import NxTableSortableExample from '../../components/NxTable/NxTableSortableExample';
```

Comment:
So you've got live examples here paired with source code displays that don't match them.  We don't do that.  The live example must be a running instance of the adjacently-displayed code.  Note that in the style examples like these, the mechanics of the example don't actually have to work (e.g. you don't have to wire up the filter inputs so that the rows actually filter) it just has to display properly.  Alternatively, you can leave the demonstration of interactive aspects like sorting and filtering to the react component page, and just document the relevant classes in the classes table on this page.  Which, admittedly, I realize we haven't done for the sorting-related classes (`.nx-cell__sort-icons`)

Reply:
Done: https://github.com/sonatype/sonatype-react-shared-components/pull/165/commits/cf17c9c6164058e69153275ca41276387bdec3bb